TCAPracticeCore/Tests/TCAPracticeCoreTests/TCAPracticeCoreTests.swift

```diff
@@ -1,12 +1,32 @@
 import XCTest
 @testable import TCAPracticeCore
+import ComposableArchitecture
 
+@MainActor
 final class TCAPracticeCoreTests: XCTestCase {
-    func testExample() throws {
-        // XCTest Documentation
-        // https://developer.apple.com/documentation/xctest
+  func testLoadAll() async throws {
+    let mocked = loadJSON()
+    let testStore = TestStore(initialState: QuizFeatureReducer.State()) {
+      QuizFeatureReducer().dependency(\.apiClient, APIClient(getAllQuestions: { mocked }))
+    }
+    await testStore.send(.loadAllQuestions)
+    await testStore.receive(.nextQuestion(nil))
```

Comment:
Don't you expect there to be a state update here? I think you should mock out your `loadJSON` dependency instead of using the live implementation because right now it's failing to open your JSON file.